modules/schematics/src/effect/index.spec.ts

```diff
@@ -250,12 +250,18 @@ describe('Effect Schematic', () => {
     expect(content).toMatch(
       /import { Actions, Effect, ofType } from '@ngrx\/effects';/
     );
+    expect(content).toMatch(/import { concatMap } from 'rxjs\/operators';/);
+    expect(content).toMatch(/import { EMPTY } from 'rxjs';/);
     expect(content).toMatch(
-      /import { FooActionTypes } from '\.\/foo.actions';/
+      /import { FooActionTypes, FooActions } from '\.\/foo.actions';/
     );
     expect(content).toMatch(/export class FooEffects/);
+    expect(content).toMatch(/loadFoos\$ = this\.actions\$.pipe\(/);
+    expect(content).toMatch(/ofType\(FooActionTypes\.LoadFoos\)/);
+    expect(content).toMatch(/concatMap\(\(\) => EMPTY\)/);
+
     expect(content).toMatch(
-      /loadFoos\$ = this\.actions\$.pipe\(ofType\(FooActionTypes\.LoadFoos\)\);/
+      /constructor\(private actions\$: Actions<FooActions>\) {}/
```

Comment:
Instead of removing the existing line, add an additional check under it.

Reply:
This is no removal, the replacement for the old line 258 is now in 259–261

Reply:
Ok, I see 👍